templates/github/.ci/scripts/check_release.py.j2

```diff
@@ -17,6 +18,17 @@ Y_CHANGELOG_EXTS = [".feature", ".removal", ".deprecation"]
 Z_CHANGELOG_EXTS = [".bugfix", ".doc", ".misc"]
 
 
+def current_version(repo):
+    try:
+        pyproject_toml = tomllib.loads(repo.git.show(f"{DEFAULT_BRANCH}:pyproject.toml"))
+        current_version = pyproject_toml["project"]["version"]
+    except Exception:
+        current_version = repo.git.grep(
+            "current_version", DEFAULT_BRANCH, "--", ".bumpversion.cfg"
+        ).split("=")[-1]
+    return Version(current_version)
```

Comment:
This implementation looks better than the one in `utils.py` (e.g the use of split+`Version`)
Any reason they are not  the same?

Reply:
Yeah, they don't run in the same context. This one reads the version from git without checking the default branch out. And it creates a `Version` object.
The other one reads the version from an actual file of the current branch while being in the plugin_template directory and we really only want the version string.